CIFAR10/train/fastai_pytorch.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students",
```

Comment:
nit: are there few enough students that you can credit them here?

Reply:
added!